pom.xml

```diff
@@ -99,6 +99,7 @@
         <module>api</module>
         <module>impl</module>
         <module>extensions</module>
+        <module>all</module>
```

Comment:
@lhazlewood `all` might not be the best directory name for this module, any alternative ideas? `legacy`, `convenience`, `the-module-formerly-known-as-jjwt`

**NOTE: ** the actual maven artifact name is `jjwt` but using that name for the directory may cause confusion between the project name.

Reply:
I updated the PR with some minor changes and changed the directory name to `tdjar` for 'transitive dependency jar' for lack of a better term.  `legacy-jar` and `convenience-jar` were too long and incongruent with the other modules.  Given that no-one except project devs will see the name, I don't think it matters that much though.